app/src/main/res/layout/fragment_forgot_password_step_two.xml

```diff
@@ -0,0 +1,78 @@
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context="com.kms.cura.view.fragment.ForgotPasswordStepTwoFragment">
+
+    <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:padding="@dimen/padding_16"
+        android:layout_marginTop="@dimen/margin_20">
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/step_2"
+            android:textSize="@dimen/textSizeTitle_25"
+            android:textColor="@color/black"
+            android:padding="@dimen/padding_5"
+            android:textStyle="bold" />
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/security_code"
+            android:textColor="@color/black"
+            android:textSize="@dimen/docText_20"
+            android:padding="@dimen/padding_5"/>
+
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="A code has been sent to your email. Please enter it below:"
```

Comment:
Put "A code has been sent to your email. Please enter it below:" in resource file